dapps/marketplace/translation/crowdin/all-messages.json

```diff
@@ -336,8 +336,6 @@
   "DM97lxszR6kNQg6jXs0vjg==": "If you have any questions or need to dispute a transaction, {E_VAR_LET_US_KNOW_B64_bGV0IHVzIGtub3c} .",
   "aOrOr85BD9OMqCaqeGLoVQ==": "let us know",
   "EbCUtytNEBvHpMUozUtwmQ==": "I got it.",
-  "L6gHSkK3U3kahjczjV1nHw==": "It seems that you are using Brave",
```

Comment:
I don't know what this is about. Maybe some code got reverted elsewhere and the translation command wasn't run? 🤷‍♂